konfera/register/views.py

```diff
@@ -21,6 +21,10 @@ def register_volunteer(request, slug):
             new_ticket.type = volunteer_ticket_type
             new_ticket.save()
 
+            if volunteer_ticket_type.accessibility != 'public':
+                volunteer_ticket_type.accessibility = 'public'
+                volunteer_ticket_type.save()
+
```

Comment:
Add message here. Have a look in events/views.py in cfp_form_view how we add messages.